velox/exec/tests/PlanBuilderTest.cpp

```diff
@@ -303,71 +304,49 @@ TEST_F(PlanBuilderTest, projectExpressions) {
 
 TEST_F(PlanBuilderTest, commitStrategyParameter) {
   auto data = makeRowVector({makeFlatVector<int64_t>(10, folly::identity)});
-  auto directory = "/some/test/directory";
+  auto outputDirectory = TempDirectoryPath::create();
```

Comment:
I don't think we need to include a dependency on `TempDirectoryPath.h` here as we don't need a valid directory for this test.

Reply:
@majetideepak Yes. Removed.